consensus/src/counters.rs

```diff
@@ -257,6 +257,69 @@ pub static CHAIN_HEALTH_PARTICIPATING_NUM_VALIDATORS: Lazy<Vec<IntGauge>> = Lazy
         .collect()
 });
 
+/// Counter for the number of blocks in the pipeline broken down by stage.
+pub static CONSENSUS_PARTICIPATION_STATUS: Lazy<IntGaugeVec> = Lazy::new(|| {
+    register_int_gauge_vec!(
+        "aptos_consensus_participation_status",
+        "Counter for consensus participation status, 0 means no participation and 1 otherwise",
+        &["peer_id"]
+    )
+    .unwrap()
+});
+
+/// For the current ordering round, voting power needed for quorum.
+pub static CONSENSUS_CURRENT_ROUND_QUORUM_VOTING_POWER: Lazy<Gauge> = Lazy::new(|| {
+    register_gauge!(
+        "aptos_consensus_current_round_total_stake",
+        "Counter for consensus participation status, 0 means no participation and 1 otherwise",
+    )
+    .unwrap()
+});
+
+/// For the current ordering round, for each peer, whether they have voted, and for which hash_index
+pub static CONSENSUS_CURRENT_ROUND_VOTED_POWER: Lazy<GaugeVec> = Lazy::new(|| {
+    register_gauge_vec!(
+        "aptos_consensus_current_round_voted_power",
+        "Counter for consensus participation status, 0 means no participation and 1 otherwise",
+        &["peer_id", "hash_index"]
```

Comment:
If the `hash_index` is not very critical for debugging, I would get rid of it as it's creating additional time series.

Reply:
it is critical to be able to see if there is a split vote (i.e. malicious actors) or not enough people active to reach consensus.

so I am not using the hash itself, but hash_index (i.e. 1,2,3,4..). without malicious actors, it should always be only 1.

Reply:
So the issue is cardinality of the series is exploded with the hash_index. If we have 100 nodes in the validator set, each node is emitting 10k time series (in the worst case) and it becomes ~1M time series across all nodes.  If the hash_index is needed, may be emit this as a separate time series and we can do join in Grafana? 

>it is critical to be able to see if there is a split vote (i.e. malicious actors) or not enough people active to reach consensus.

Can you clarify how its helpful in  detecting malicious votes? IIUC, any double voting is being rejected without the counters being affected today.